src/index.js

```diff
@@ -24,9 +25,10 @@ export {
   List,
   ListItem,
   Checkbox,
-  Progressbar,
+  Pagination,
```

Comment:
also, put this in the component. pagination is not implementation of native dom element, and it is interact with state change. It is better fit to component.